models/intermediate/field_history/int_jira__timestamp_field_history_scd.sql

```diff
@@ -0,0 +1,57 @@
+{%- set base_columns = ['updated_at', 'issue_id', 'updated_at_week', 'status', 'author_id'] -%}
+{%- set custom_columns = [] -%}
+{%- for col in var('issue_field_history_columns', []) -%}
+    {%- set clean_col = dbt_utils.slugify(col) | replace(' ', '_') | lower -%}
+    {%- set _ = custom_columns.append(clean_col) -%}
```

Comment:
Set being used in this way is technically not the right way to do it. To ensure proper support we need to leverage `do`.
```suggestion
    {%- do custom_columns.append(clean_col) -%}
```